js/src/pages/shipping/index.js

```diff
@@ -37,7 +37,7 @@ import { recordGlaEvent } from '~/utils/tracks';
  * Page component to edit audience and shipping settings.
  *
  * Note that:
- * - This page used to be called "Edit free listings" page.
+ * - This page used to be called "Edit product feed" page.
```

Comment:
Since this code comment explains the historical context, it might be worth keeping unchanged.